### PR TITLE
AUTH_ENABLED removal

### PR DESCRIPTION
**Pull request by valentinyanakiev**
Removed AUTH_ENABLED dependencies.

Requires to be tested before promoting the PR.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -107,14 +123,13 @@ services:
 
   alkemio_server:
     container_name: alkemio_demo_server
-    image: alkemio/server:v0.12.5p1
+    image: vyanakiev/alkemio-server:v0.14.2
```
Comment (techsmyth) [inline docker-compose.yaml]: To be fixed before merging

**Review by techsmyth — APPROVED** [review]
Could also have this PR merge into the main demo refresh PR?

File: docker-compose.yaml (hunk 1 above)
Comment (valentinyanakiev) [reply docker-compose.yaml]: Also need to merge the other branch...

**Comment by valentinyanakiev** [comment]
Population works with the latest changes now - we just need to update client + server dependencies.

**Pull request merged**